src/contrib/cluster/Akka.Cluster.Tools/PublishSubscribe/DistributedPubSubMediator.cs

```diff
@@ -500,7 +500,7 @@ private void IgnoreOrSendToDeadLetters(object message)
                 Context.System.DeadLetters.Tell(new DeadLetter(message, Sender, Context.Self));
         }
 
-        private void PublishMessage(string path, object message, bool allButSelf = false)
+        private void PublishMessage(string path, IWrappedMessage publish, bool allButSelf = false)
```

Comment:
Rather than send just the `object Message` to these methods, send the full `IWrappedMessage` payload which we'll pass directly to the `IgnoreOrSendToDeadLetters` method - as that wrapper message includes all of the useful debugging context a developer might need.